exonum-java-binding/CHANGELOG.md

```diff
@@ -19,7 +19,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Overview
 
-This release brings support of Exonum TestKit. It is based on Exonum 0.11.
+This release brings support of Exonum TestKit, massive performance improvements,
+and various other fixes and improvements. It is based on Exonum 0.11.
+
+*If you are upgrading an existing Java service, consult 
+the [migration guide](https://github.com/exonum/exonum-java-binding/blob/ejb/v0.7.0/exonum-java-binding/doc/Migration_guide_0.7.md).*
```

Comment:
wouldn't it be better to have a relative link like `./doc/Migration_guide_0.7.md`?

Reply:
Definitely (when viewed offline), but it is also used as a source for the release notes, where such links won't work.